2025-08-06 22:30:29,629 - app.api.routes.query - ERROR - Erreur lors de la requête RAG: 'TokenData' object has no attribute 'get'
2025-08-06 22:33:07,374 - app.api.routes.query - ERROR - Erreur lors de la requête RAG: 'TokenData' object has no attribute 'get'
2025-08-06 23:15:45,124 - app.api.routes.document - ERROR - Erreur lors de la suppression: 'TokenData' object has no attribute 'get'
2025-08-06 23:18:44,600 - app.api.routes.document - ERROR - Erreur lors de la suppression: 'TokenData' object has no attribute 'get'
2025-08-06 23:20:40,297 - app.api.routes.document - ERROR - Erreur lors de la suppression: 'TokenData' object has no attribute 'sub'
2025-08-06 23:22:53,796 - app.api.routes.document - ERROR - Erreur lors de l'upload: 'TokenData' object has no attribute 'get'
2025-08-06 23:46:23,455 - app.services.implementations.llm_service - ERROR - Erreur lors de l'inférence LLM: 402 Client Error: Payment Required for url: https://router.huggingface.co/novita/v3/openai/chat/completions (Request ID: Root=1-6893db40-51a16741565245ea77f264f2;dfc059bd-fd7a-4d00-a4c3-04edb050b009)

You have exceeded your monthly included credits for Inference Providers. Subscribe to PRO to get 20x more monthly included credits.
2025-08-19 10:26:19,178 - app.services.implementations.llm_service - ERROR - Erreur lors de l'inférence LLM: 402 Client Error: Payment Required for url: https://router.huggingface.co/novita/v3/openai/chat/completions (Request ID: Root=1-68a4433b-5bfc06eb43da71f124304623;0dafca5a-fd3c-4dd5-9189-5b74ad509e5f)

You have exceeded your monthly included credits for Inference Providers. Subscribe to PRO to get 20x more monthly included credits.
2025-08-21 20:08:00,965 - app.services.implementations.document_service - ERROR - Erreur lors du traitement du document: PDFMetadata.__init__() missing 1 required positional argument: 'file_size'
2025-08-21 20:18:17,602 - app.services.implementations.document_service - ERROR - Erreur lors du traitement du document: PdfProcessor.vectorize_text() missing 1 required positional argument: 'file_size'
2025-08-27 10:42:04,687 - app.services.implementations.local_llm_service - ERROR - Erreur lors du chargement du modèle: Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, '-' and '.' cannot start or end the name, max length is 96: 'C:\Users\HP\Documents\INDIA-2\stage-repo\ChatBot-RAG\chat-backend\app\llms_models\gpt2'.
2025-08-27 10:46:30,388 - app.services.implementations.local_llm_service - ERROR - Erreur lors du chargement du modèle: Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, '-' and '.' cannot start or end the name, max length is 96: 'C:\Users\HP\Documents\INDIA-2\stage-repo\ChatBot-RAG\chat-backend\app\llms_models\gpt2'.
2025-08-27 10:46:30,390 - app.services.startup_service - ERROR -  Erreur lors de l'initialisation des services: Impossible de charger le modèle: Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, '-' and '.' cannot start or end the name, max length is 96: 'C:\Users\HP\Documents\INDIA-2\stage-repo\ChatBot-RAG\chat-backend\app\llms_models\gpt2'.
2025-08-27 10:49:45,936 - app.services.implementations.local_llm_service - ERROR - Erreur lors du chargement du modèle: Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, '-' and '.' cannot start or end the name, max length is 96: 'C:\Users\HP\Documents\INDIA-2\stage-repo\ChatBot-RAG\chat-backend\app\llms_models\gpt2'.
2025-08-27 10:49:45,938 - app.services.startup_service - ERROR -  Erreur lors de l'initialisation des services: Impossible de charger le modèle: Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, '-' and '.' cannot start or end the name, max length is 96: 'C:\Users\HP\Documents\INDIA-2\stage-repo\ChatBot-RAG\chat-backend\app\llms_models\gpt2'.
2025-08-27 10:51:42,078 - app.services.implementations.local_llm_service - ERROR - Erreur lors du chargement du modèle: Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, '-' and '.' cannot start or end the name, max length is 96: 'C:\Users\HP\Documents\INDIA-2\stage-repo\ChatBot-RAG\chat-backend\app\llms_models\gpt2'.
2025-08-27 10:51:42,080 - app.services.startup_service - ERROR -  Erreur lors de l'initialisation des services: Impossible de charger le modèle: Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, '-' and '.' cannot start or end the name, max length is 96: 'C:\Users\HP\Documents\INDIA-2\stage-repo\ChatBot-RAG\chat-backend\app\llms_models\gpt2'.
2025-08-27 10:57:12,574 - app.services.implementations.local_llm_service - ERROR - Erreur lors du chargement du modèle: Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, '-' and '.' cannot start or end the name, max length is 96: 'C:\Users\HP\Documents\INDIA-2\stage-repo\ChatBot-RAG\chat-backend\app\llms_models\gpt2'.
2025-08-27 10:57:12,576 - app.services.startup_service - ERROR -  Erreur lors de l'initialisation des services: Impossible de charger le modèle: Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, '-' and '.' cannot start or end the name, max length is 96: 'C:\Users\HP\Documents\INDIA-2\stage-repo\ChatBot-RAG\chat-backend\app\llms_models\gpt2'.
2025-08-27 11:00:15,689 - app.services.implementations.local_llm_service - ERROR - Erreur lors du chargement du modèle: Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, '-' and '.' cannot start or end the name, max length is 96: 'C:\Users\HP\Documents\INDIA-2\stage-repo\ChatBot-RAG\chat-backend\app\llms_models\gpt2'.
2025-08-27 11:00:15,693 - app.services.startup_service - ERROR -  Erreur lors de l'initialisation des services: Impossible de charger le modèle: Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, '-' and '.' cannot start or end the name, max length is 96: 'C:\Users\HP\Documents\INDIA-2\stage-repo\ChatBot-RAG\chat-backend\app\llms_models\gpt2'.
2025-08-27 11:02:42,614 - app.services.implementations.local_llm_service - ERROR - Erreur lors du chargement du modèle: Le répertoire du modèle n'existe pas: C:\Users\HP\Documents\INDIA-2\stage-repo\ChatBot-RAG\chat-backend\app\llms_models\gpt2
2025-08-27 11:02:42,616 - app.services.startup_service - ERROR -  Erreur lors de l'initialisation des services: Impossible de charger le modèle: Le répertoire du modèle n'existe pas: C:\Users\HP\Documents\INDIA-2\stage-repo\ChatBot-RAG\chat-backend\app\llms_models\gpt2
2025-08-27 11:19:42,233 - app.services.startup_service - ERROR -  Erreur lors de l'initialisation des services: 'model_path'
2025-08-27 11:22:25,664 - app.services.startup_service - ERROR -  Erreur lors de l'initialisation des services: 'model_path'
2025-08-27 11:41:10,228 - app.services.implementations.local_llm_service - ERROR - Erreur lors de l'infÃ©rence LLM: index out of range in self
2025-08-27 11:42:51,483 - app.services.implementations.local_llm_service - ERROR - Erreur lors de l'infÃ©rence LLM: index out of range in self
2025-08-27 11:45:05,233 - app.services.implementations.local_llm_service - ERROR - Erreur lors de l'infÃ©rence LLM: index out of range in self
